js/interface.js

```diff
@@ -56,7 +60,7 @@ var getApps = Fliplet.Apps.get().then(function(apps) {
 
 var widgetId = parseInt(Fliplet.Widget.getDefaultId(), 10);
 var widgetData = Fliplet.Widget.getData(widgetId) || {};
-
+const appId = widgetData.appId;
```

Comment:
What is the purpose of this 'const appId'

Reply:
we can not pass the widgetData.appId in object of Fliplet.App.Tokens.get({**widgetData.appId**, query: {type: 'integrationToken'}}),
it will show the parsing error.

Reply:
@amolsuryawanshi47 of course that shows a parsing error, it's not proper Javascript.

This is not a valid syntax:

```js
{ foo.bar }
```

On the other hand, this is:

```js
{ appId: widgetData.appId }
```

Reply:
Yes @squallstar you right. It's my bad.